subsys/shell/Kconfig.backends

```diff
@@ -56,14 +56,85 @@ config SHELL_BACKEND_SERIAL_RX_POLL_PERIOD
 	help
 	  Determines how often UART is polled for RX byte.
 
+choice
+	prompt "Initial log level limit"
+	default SHELL_BACKEND_SERIAL_LOG_LEVEL_DEFAULT
+
+config SHELL_BACKEND_SERIAL_LOG_LEVEL_DEFAULT
+	bool "System limit (LOG_MAX_LEVEL)"
+	
+config SHELL_BACKEND_SERIAL_LOG_LEVEL_DBG
+	bool "Debug"
+	
+config SHELL_BACKEND_SERIAL_LOG_LEVEL_INF
+	bool "Info"
+	
+config SHELL_BACKEND_SERIAL_LOG_LEVEL_WRN
+	bool "Warning"
+	
+config SHELL_BACKEND_SERIAL_LOG_LEVEL_ERR
+	bool "Error"
+	
```

Comment:
Here's for example trailing whitespace (on otherwise empty line) about  which checkpatch complains (and on bunch of other lines too).